gmso/formats/mcf.py

```diff
@@ -0,0 +1,700 @@
+from __future__ import division
+
+import warnings
+import datetime
+from math import sqrt
+
+import numpy as np
+import networkx as nx
+import sympy
+import unyt as u
+
+from gmso.core.topology import Topology
+from gmso.lib.potential_templates import LennardJonesPotential
+from gmso.lib.potential_templates import MiePotential
+from gmso.lib.potential_templates import HarmonicAnglePotential
+from gmso.lib.potential_templates import HarmonicTorsionPotential
+from gmso.lib.potential_templates import PeriodicTorsionPotential
+from gmso.lib.potential_templates import OPLSTorsionPotential
+from gmso.lib.potential_templates import RyckaertBellemansTorsionPotential
+from gmso.lib.potential_templates import HarmonicImproperPotential
```

Comment:
This criticism is not at all directed at you, I just want to note it somewhere - long lists of imports is a hint that we should think about doing this in a different way